samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobVideoClassificationSampleTest.java

```diff
@@ -34,7 +34,7 @@
 public class CreateBatchPredictionJobVideoClassificationSampleTest {
 
   private static final String PROJECT = System.getenv("UCAIP_PROJECT_ID");
-  private static final String MODEL_ID = System.getenv("BATCH_PREDICTION_VIDEO_CLASS_MODEL_ID");
+  private static final String MODEL_ID = "8596984660557299712";

```

Comment:
why do we need to hardcode this? can we just update the value for this env var in secret manager?

Reply:
Acknowledged. 

This is how we're currently managing resource IDs for nodejs-ai-platform and python-aiplatform. We may return to using Secret Manager once we've arrived at a decision by the team.